### DISCUSSION TITLE
YoloV5 - Draw Motion Path

### DISCUSSION THREAD

**Discussion by jacobchikku** [ultralytics/yolov5#3503] [Q&A]
![Tracking](https://user-images.githubusercontent.com/8341121/121006624-b6cc2080-c7ae-11eb-9afe-ab81ec4f9a40.jpg)
I am trying to Draw a line to identify the Motion Path for the tracked objects, is it possible to use Yolov5.

As shown in the above picture, once identifying the ball, I would like to display a trailing line connecting all the points through which it was traveling. 

The motion path line needs to be displayed until the object is out of focus in the video.  What is the best option to get it done? Can anyone please advise?  

Many Thanks

**Comment by glenn-jocher** [comment]
@jacobchikku yes you can do that! For an example see:  https://www.reddit.com/r/compsci/comments/navkf0/oc_multiple_object_tracking_with_yolov5_and_opencv/

<img width="779" alt="Screenshot 2021-06-07 at 13 09 46" src="https://user-images.githubusercontent.com/26833433/121006843-a6597d00-c791-11eb-9d9f-0863dbf7f411.png">

**Reply by jacobchikku** [reply]
Thanks @glenn-jocher 

Yes, this is exacting what I am looking for, Glad to see it's possible in Yolov5.

I am extremely new to YoloV5. It will be of great help if I could get a code sample to see how the points are connected and a line is drawn connecting them.

**Reply by jacobchikku** [reply]
I have tried multiple ways to get previous detection points or id, but I am not sure how to get the previous detections and how to connect them to draw the line.

**Reply by jacobchikku** [reply]
hey @glenn-jocher, is there any code sample for the same? Connecting the previous detection points of an object and drawing a line?

**Reply by glenn-jocher** [reply]
@jacobchikku I would contact the authors of the above work

**Reply by jacobchikku** [reply]
@glenn-jocher Thank you so much

**Reply by jacobchikku** [reply]
Hi @glenn-jocher, any luck we have?

**Reply by jacobchikku** [reply]
I have been trying but not yet successful.
